docs/standard/attributes/writing-custom-attributes.md

```diff
@@ -36,27 +36,27 @@ To design your own custom attributes, you do not need to master many new concept
   
 <a name="cpconapplyingattributeusageattribute"></a>   
```

Comment:
do you need these anchors? You could use the regular Markdown anchors on the introduction no?